client/src/components/Voting.tsx

```diff
@@ -0,0 +1,290 @@
+import React, { useCallback, useEffect, useState } from "react";
+import axios from "axios";
+
+import { DragDropContext, Droppable, Draggable, DropResult } from 'react-beautiful-dnd'
+import { Selector } from 'tabler-icons-react'
+import { useForm } from "@mantine/form"
+import { useListState } from "@mantine/hooks";
+import { Button, createStyles, TextInput } from "@mantine/core";
+
+import { Election } from "./Election";
+import useAuthorization from "../hooks/useAuthorization";
+import constants from "../util/constants";
+import { compareList } from "../util/funcs";
+import { ErrorModal, InformationModal } from "./Information";
+
+interface candidateInfo {
+  id: string,
+  name: string,
+  presentation: string, 
+  symbolic: boolean
+}
+
+const useStyle = createStyles((theme) => ({
+  item: {
+    display: 'flex',
+    alignItems: 'center',
+    borderRadius: theme.radius.md,
+    border: `1px solid ${theme.colors.gray[4] }`,
+    padding: `${theme.spacing.sm}px ${theme.spacing.xl}px`,
+    backgroundColor: theme.white,
+    marginBottom: theme.spacing.sm,
+    top: "auto!important",
+    left: "auto!important",
+  },
+
+  droppable: {
+    // overflowY: "scroll",
+  },
+
+  dragHandle: {
+    ...theme.fn.focusStyles(),
+    display: 'flex',
+    alignItems: 'center',
+    height: '100%',
+    padding: "1rem",
+    paddingTop: "2rem",
+    margin: "-1rem",
+    marginLeft: "-2rem",
+    marginTop: "-2rem"
```

Comment:
you can do `padding: TOP RIGHT BOTTOM LEFT` to specify different values for different sides in one line